backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -31,6 +31,11 @@ body {
     @include margin(null auto);
     max-width: map-get($container-max-widths, "xl");
   }
+
+  &.left-aligned {
```

Comment:
I'm guessing the purpose of this is to add the `max-width` to it? I wonder if we can come up with a different name as everything* is left aligned by default.

*almost everything